identity/handler.go

```diff
@@ -39,6 +39,16 @@ const (
 	BatchPatchIdentitiesLimit = 2000
 )
 
+// fandom-start - add API to validate email before saving user in UCP
+const RouteValidate = RouteCollection + "/validate"
+
+// fandom-end
+
+// fandom-start allow admins to remove multifactor authentication for an identity
+const RouteMultifactor = RouteItem + "/credentials/:credentialType"
+
+// fandom-end
+
```

Comment:
Nitpick (codestyle): Let's add these values to `const` block above

Reply:
Fixed.